src/core/Akka/Dispatch/AbstractDispatcher.cs

```diff
@@ -323,6 +446,14 @@ protected void ReportFailure(Exception ex)
         /// </remarks>
         protected abstract void Shutdown();
 
+        /// <summary>
+        /// Testing purposes only
+        /// </summary>
+        internal void ShutdownInternal()
```

Comment:
**Possible Fix**: Just realized I might be able to remove this... Will need to double check.